connectivity/mbedtls/tools/importer/Makefile

```diff
@@ -31,13 +31,13 @@ MBED_TLS_RELEASE ?= mbedtls-2.22.0
 MBED_TLS_REPO_URL ?= git@github.com:ARMmbed/mbedtls.git
 
 # Translate between mbed TLS namespace and mbed namespace
-TARGET_PREFIX:=../
```

Comment:
The mbedtls tag at line 30 says 2.22. It should be updated to 2.24

Reply:
It is 2.24.0, if you see the next commit